test/Recipe-test.js

```diff
@@ -125,7 +57,71 @@ describe('Recipe Class', function() {
       ]
     };
 
-    recipe = new Recipe(recipeDatum);
+    recipe = new Recipe(recipeDetails);
+  });
+
+  it('should have be a function', function() {
+    expect(Recipe).to.be.a('function');
   });
 
+  it('should be an instance of Recipe', function() {
+    expect(recipe).to.be.an.instanceOf(Recipe);
+  });
+
+  desbribe('Recipe constructor properties', function() {
```

Comment:
I could be wrong, but don't we need to instantiate the Recipe class within each ```it```block?